### DISCUSSION TITLE
Release notes?

### DISCUSSION THREAD

**Discussion by lanesawyer** [chanzuckerberg/sci-components#785] [General]
I'm getting an error when using `ButtonDropdown` ("Warning: React does not recognize the `sdsStyle` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `sdsstyle` instead. If you accidentally passed it from a parent component, remove it from the DOM element."). I noticed there's a newer version (we're on `v20.4.0`) and was curious on if the latest package fixed it, but I can't find release notes anywhere to confirm.

Are those in a place I just haven't looked yet? I don't see anything on `npmjs.org`, in the GitHub Releases that are published, or on the SDS documentation site.

If those don't exist yet, adding release notes would be very helpful to library consumers!

**Comment by masoudmanson** [comment]
Hey Lane,

Thanks for reporting the bug. I believe the newer version has resolved this particular issue you're having. Please try `@czi-sds/components@20.5.0` and let us know if it works as expected. We have refactored the Button components in this version.

As for the release notes, we have some notes written inside the [changelog file](https://github.com/chanzuckerberg/sci-components/blob/main/packages/components/CHANGELOG.md), but we are considering more structured release notes in the future. Sorry for any inconvenience.

**Reply by lanesawyer** [reply]
Sweet, we'll try the upgrade for the bug, and glad to hear you're considering more robust release notes. On the Allen Institute Brain Knowledge Platform we use `git-cliff` to auto-publish our conventional commits (which I see you're already using 🎉).

And I don't know why I neglected to look for `CHANGELOG.md` lol 🤦🏻

Thanks!
